src/dom-utils.ts

```diff
@@ -88,6 +110,14 @@ export function isFocusable(el: HTMLElement) {
   )
 }
 
+function canHaveFocusableChildren(el: HTMLElement) {
```

Comment:
Can we have a small comment to explain the goal of that function?

Reply:
On it!